src/sentry/grouping/component.py

```diff
@@ -35,26 +35,26 @@ class GroupingComponent:
     into components to make a hash for grouping purposes.
     """
 
+    id: str = "default"
```

Comment:
why does this need a default if `id` is required in the init?

Reply:
You're right, at this point it doesn't. For simplicity I included the default value here (where I was setting other defaults) rather than in https://github.com/getsentry/sentry/pull/80722, which is where it actually comes into play. But given our conversation there about not passing in `id` at all, it's about to become a moot point.